src/d3a/d3a_core/redis_connections/aggregator_connection.py

```diff
@@ -49,6 +51,87 @@ def add_batch_trade_event(self, device_uuid, event):
     def add_batch_finished_event(self, device_uuid, event):
         self._add_batch_event(device_uuid, event, self.batch_finished_events)
 
+    def crud_aggregator_callback(self, payload):
```

Comment:
Also here, please remove all "crud" occurrences.  create, read, update, and delete does not apply here, because the aggregator is not some storage. IMO it is confusing and should be renamed. Thanks